model/generate.go

```diff
@@ -170,9 +170,9 @@ func (g *GeneratedProject) Save(ctx context.Context, p *Project, pp *ParserProje
 
 // update the parser project using the newest config number (if using legacy version config, this comes from version)
 func updateParserProject(v *Version, pp *ParserProject) error {
-	updateNum := pp.ConfigUpdateNumber + 1
+	updateNum := *pp.ConfigUpdateNumber + 1
```

Comment:
It's possible that ConfigUpdateNumber will be nil so you can't do this directly, so we have utility functions for this (utility.FromIntPtr)